cypress/pageobject/Patient/PatientDetails.ts

```diff
@@ -0,0 +1,46 @@
+export class PatientDetailsPage {
+  clickAssignToVolunteer() {
+    cy.contains("button", "Assign to a Volunteer")
+      .scrollIntoView()
+      .should("be.visible")
+      .should("be.enabled")
+      .click();
+  }
+
+  selectAndAssignVolunteer(volunteerName: string) {
+    cy.clickAndSelectOption("#assign_volunteer", volunteerName);
+    cy.clickSubmitButton("Assign");
+    cy.get("#assigned-volunteer", { timeout: 10000 })
+      .scrollIntoView()
+      .should("be.visible")
+      .should("contain.text", volunteerName);
+  }
+
+  verifyVolunteerBannerIsUpdated(volunteerName: string) {
+    cy.get("#assigned-volunteer").should(
```

Comment:
```suggestion
    cy.get("#assigned-volunteer").should(
```

check the function cy.verifyContentPresence